neutrinoapp/src/Search.jsx

```diff
@@ -144,7 +150,7 @@ export default function Search () {
         <h1>OHOS</h1>
         <table className='table'>
           <tbody>
-            {
+            { !error400 ?
```

Comment:
If I'm reading this right, it says "if not error400 do this; else do that". I think "if error400 do that; else do this" is a touch easier to understand. But this is just a suggestion.

Reply:
I was keeping the non error state first but I have rearranged it.